web-api-shared/src/elastic.rs

```diff
@@ -219,35 +247,144 @@ impl<I> BulkResponse<I> {
 }
 
 #[derive(Debug, Deserialize)]
-struct Hit<I, T> {
+struct Hit<I> {
     #[serde(rename = "_id")]
     id: I,
-    #[allow(dead_code)]
-    #[serde(rename = "_source")]
-    source: T,
```

Comment:
This one was a "fun" bug.

We do not include `_source` anymore in any request, so no matter what `T` serializes to de-serializing this should always have failed. 

But it didn't.

It turns out that somehow the combinations of `serde_json` with a `T::deserialize` which never calls the deserializers somehow "eats" the bug that the filed is fully missing.

Be aware that serde requires the deserializer to be called, so this wasn't a bug in `serde`/`serde_json` but our code.